docs/tempo/website/getting-started/traces.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
+- /docs/tempo/getting-started/traces
+description: "What are traces?"
+keywords:
+- Grafana
+- traces
+- tracing
+title: What are traces?
+weight: 120
+---
+
+# What are traces?
+
+A traces represents the whole journey of a request or an action as it moves through all the nodes of a distributed system, especially containerized applications or microservices architectures. Traces let you profile and observe systems, making it easy to discover bottlenecks and interconnection issues.
+
+## Example of traces
+
+A user on your website enters their email address into a form to sign up for your mailing list. They click **Enter**. This initial transaction has a trace ID which is subsequently associated with every interaction in the chain of processes within a system.
```

Comment:
I think it's important we explain the concept of `context propagation` as the base of tracing in distributed systems. Maybe here or in the terminology section.